src/core/extendRoutes.spec.ts

```diff
@@ -251,4 +251,26 @@ describe('EditableTreeNode', () => {
       },
     ])
   })
+
+  it('can override the path', () => {
+    const tree = new PrefixTree(RESOLVED_OPTIONS)
+    const editable = new EditableTreeNode(tree)
+    const node = editable.insert('foo', 'file.vue')
+    const childNode = node.insert('bar', 'file.vue')
+
+    childNode.path = 'baz'
+    node.path = 'baz'
+    editable.path = 'baz'
```

Comment:
FYI: the root node shouldn’t be changed. Or at least not for this

Reply:
Great! I'll update that now 😊